browse-scaler/src/index.js

```diff
@@ -13,54 +15,82 @@ const buildResponse = (image, format) => {
   };
 };
 
-const getImageUrlFromConcept = async conceptId => {
-  if (conceptId.startsWith('G')) {
+const getImageUrlFromConcept = async (conceptId, conceptType) => {
+  console.log(`Concept id: ${conceptId}`);
+  if (conceptId === null || conceptId === undefined) {
+    return null;
+  }
+
+  if (conceptType === 'granules') {
     return getGranuleLevelBrowseImage(conceptId);
   }
 
-  if (conceptId.startsWith('C')) {
+  if (conceptType === 'datasets') {
     return getCollectionLevelBrowseImage(conceptId);
   }
 
   console.log(`Unable to find browse imagery for concept: ${conceptId}`);
   return null;
 };
 
-const resizeImageFromConceptId = async (conceptId, height, width) => {
+const resizeImageFromConceptId = async (conceptType, conceptId, height, width) => {
   // If given an image url, fetch the image and resize. If no image
   // exists, return the not found response
-  const imageUrl = await getImageUrlFromConcept;
+  const imageUrl = await getImageUrlFromConcept(conceptId, conceptType);
 
   if (imageUrl === null) {
-    const cachedSvg = await getImageFromCache('NOT-FOUND');
-    if (cachedSvg) {
-      return buildResponse(cachedSvg, 'image/svg');
-    }
+    // const cachedSvg = await getImageFromCache("NOT-FOUND");
+    // if (cachedSvg) {
+    //   return buildResponse(cachedSvg, "image/svg");
+    // }
 
-    const imgNotFound = notFound(height, width);
-    cacheImage('NOT-FOUND', imgNotFound);
+    const imgNotFound = await notFound(height, width);
+    // cacheImage("NOT-FOUND", imgNotFound);
 
     return buildResponse(imgNotFound, 'image/svg');
   }
 
-  const cacheKey = `${conceptId}-${height}-${width}`;
-  const imageFromCache = await getImageFromCache(cacheKey);
-  if (imageFromCache) {
-    return buildResponse(imageFromCache, 'image/png');
-  }
+  // CMR-5774: This is commented out while we work on fixing the SG that
```

Comment:
Note that there's also commented out code above - maybe move this comment up higher if it also applies to that code as well.